ozaria/site/components/interactive/PageInteractive/draggableOrdering/index.vue

```diff
@@ -139,17 +134,38 @@
     align-items: center;
     justify-content: space-evenly;
 
-    /deep/ .draggable-slot {
-      height: 53px;
-      border: 1px solid black;
+    li {
+      margin-bottom: 15px;
 
-      padding: 0;
-      list-style: none;
       width: 100%;
-      li {
-        text-align: center;
-      }
+
+      display: flex;
+
+      justify-content: center;
+      align-items: center;
+
+      text-align: center;
+      font-size: 15px;
+
+      min-height: 50px;
+    }
+
+    li.prompt {
+      border: 2px solid #acb9fa;
+    }
+
+    li.prompt-label {
+      background-color: #acb9fa;
+      border: 2px solid #acb9fa;
+    }
+
+    li.monospaced {
+      font-family: monospace;
     }
-  }
 
+    li.dragging-slot {
+      // TODO this doesn't work because vue-draggable also uses transforms for positioing
```

Comment:
TODO left here?

Reply:
Unfortunately this is still valid and will need to be handled in UX. vue-draggable uses transforms to execute the drag functionality and CSS transforms are not additive, so we'll need to figure out a different way to handle this.  We may need to target an element below `dragging-slot`.  Not sure if this is possible to accomplish and maintain proper flexbox layout / sizing.